conda_content_trust/cli.py

```diff
@@ -24,6 +25,17 @@
 
 
 def cli(args=None):
+    parser = build_parser()
+
+    args = parser.parse_args(args)
+
+    if hasattr(args, "func"):
+        return args.func(args)
+    else:
+        parser.print_help()
```

Comment:
can be simplified by setting `sp.required = True` in line 40/52 and then just
```suggestion
    parser.print_help()
```

Reply:
You're saying for all the subparsers, I need to add a required=True parameter?

Reply:
I added required=True and change `subcommand_name` to `subcommand`, to produce this error message when no subcommand is given `conda-content-trust: error: the following arguments are required: subcommand`

Reply:
yep adding `required=True` once as you did does the trick